pkg/service/delete.go

```diff
@@ -66,6 +66,16 @@ func (c *DeleteCommand) Exec(in io.Reader, out io.Writer) error {
 		}
 	}
 
+	if err := c.manifest.File.Read(manifest.Filename); err != nil {
+		return fmt.Errorf("error reading package manifest: %w", err)
```

Comment:
Would this not fail when running delete outside of a project directory? I would expect the command to still work for normal VCL services or via passing in a flag

Reply:
Ooh good point. Let me have a think about the implementation 👍🏻